### ISSUE TITLE
Allow admins to customize application completion text

### ISSUE THREAD

**Issue by azizova-leyla** [seattle-uat/civiform#2892]
**Is your feature request related to a problem? Please describe.**
Residents want more information about next steps after they submit applications. This needs to be program specific.

**Describe the solution you'd like**
Allow admins to write custom text for that page.

**Additional context**
This text:
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/1741747/178819014-5d17231c-e277-4f2b-a24d-b67aad06e276.png">

**Done when**
What conditions need to be met before this issue can be closed?

**Comment by rahaghas** [comment]
Here is a mock example of where we could show that there are next steps for a given program (ideally this would be combined with an email notification):
<img width="1069" alt="Screen Shot 2022-07-18 at 11 57 05 AM" src="https://user-images.githubusercontent.com/103459221/179552800-b97764d0-f329-4878-b10b-c8bf270362f5.png">

Clicking "Next steps" here would lead applicants to a new page with content created by the program admin. On the program admin side, we could offer this feature within the status tracking experience. One question I have is whether "Next steps" would only be available for approved applications or just all applications that are submitted?

**Comment by azizova-leyla** [comment]
The use case I was thinking about it for an applicant to have this information right after they submit, before the status is knows. Things to include there:
1. Timeframe for the decision to be known
2. How the decision will be shared with them (will they get email? do they need to come back to civiform? will it be automatically applied to their bill?)
3. Phone/email they can use to contact Program owners to ask clarifying questions

This was from UXR we did in Bloomington: 
![image](https://user-images.githubusercontent.com/1741747/179560344-56d8f290-bb12-420f-9fa9-005e7940883c.png)

**Comment by rahaghas** [comment]
Gotcha! How's something like this?
<img width="1184" alt="Screen Shot 2022-07-18 at 2 08 04 PM" src="https://user-images.githubusercontent.com/103459221/179574986-4db29720-307e-4783-b377-86b88fe1088b.png">

**Comment by azizova-leyla** [comment]
Yes, that would work. Do we intend applicants to be redirected here after they complete the application? I think it's important for them to see this right after they hit submit.

**Comment by rahaghas** [comment]
We can also put this information in the confirmation page, but this shows up in their program dashboard.

**Comment by rockycodes** [comment]
This is done: https://github.com/civiform/civiform/pull/4264
